### PR TITLE
Merchant invoice total revenue

### LINKED ISSUE #24: Merchant Invoice Show Page: Total Revenue
As a merchant
When I visit my merchant invoice show page
Then I see the total revenue that will be generated from all of my items on the invoice

### PR DESCRIPTION
**Pull request by JoePeterson51**
### Features being added/updated
Merchant invoice total revenue

### Any issues this PR will unblock/block (Leave blank if no blockers are involved)

  
- [X] Ready for merge (Don't merge unless the author of the PR has checked this)
- [X] All Tests are passing

(Link issue number next to `#` to automatically move the card on kanban board)

closes #24

### CODE REVIEW

**Review by denverdevelopments — COMMENTED** [review]
It looks good, but I notice there's only a 'binding.pry' in the body of method 'def self.total_revenue_from' method in invoice.rb.

File: app/models/invoice.rb [hunk 1]
```diff
@@ -14,6 +14,13 @@ def self.invoices_for_merchant(merchant_id)
     Invoice.joins(:items).where(items: {merchant_id: merchant_id})
   end
 
+  def self.total_revenue_from(invoice_items)
+    
+    binding.pry
+
```
Comment (redferret) [inline app/models/invoice.rb]: may want to remove this pry as well as the method to clean it up some

**Review by redferret — APPROVED** [review]
Looks good but just keep this in mind `invoice_items.sum('invoice_items.unit_price * invoice_items.quantity')`

**Review by JoannaCoPo — APPROVED** [review]
Well done, sir.

**Review by JoannaCoPo — APPROVED** [review]

**Pull request merged**